src/calculateRentalCost.js

```diff
@@ -5,6 +5,19 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days === 7) {
+    return 230;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 3) {
+    return 100;
+  }
+
+  return 80;
```

Comment:
The function uses hardcoded values for the discount logic. According to the checklist, you should avoid using literals in expressions and instead use constants that explain these values. You can define constants for the daily cost, long-term rental threshold and discount, and short-term rental threshold and discount, then use these constants within your logic.